src/agent/agent.c

```diff
@@ -30,6 +31,69 @@ static bool
                                 void *userdata,
                                 free_func_t free_userdata);
 
+#define AGENT_HEARTBEAT_INTERVAL_USEC (1000000)
```

Comment:
I'll change this to 20 ms per the discussion.

Reply:
It should def be a config option